src/gpiocmds.c

```diff
@@ -72,10 +115,13 @@ void
 command_update_digital_out(uint32_t *args)
 {
     struct digital_out_s *d = oid_lookup(args[0], command_config_digital_out);
-    sched_del_timer(&d->timer);
+    //FIXME: How to act here if queue is still populated?
```

Comment:
Currently, either update_digital_out is used exclusively, or scheduled events are used exclusively.  So, I'd call `shutdown()` if update_digital_out is called when events are scheduled.